Documentation/image-fetching-behavior.md

```diff
@@ -0,0 +1,27 @@
+# Image fetching behavior
+
+When fetching, rkt will try to avoid unnecessary network transfers: if an updated image is already in the local store there's no need to download it again.
+
+This behavior can be controlled with the `--store-only` and `--no-store` flags.
+
+## General behavior
+
+--store-only | --no-store |  behavior          | description
+ ----------- | ---------- | ------------------ | -------------------------------------------------------------------------------
+1            | 0          | store              | Just check the store
```

Comment:
I think we could make this a little more user-friendly:
- "set" / "unset" (or "true" / "false") instead of 1 / 0
- perhaps "Fetch from..." instead of "behavior"?
- a more verbose description of the tables